docs/clients/mpd.rst

```diff
@@ -167,5 +167,5 @@ projects are a real match made in heaven."
 Partify
 -------
 
-`Partify <https://github.com/fhats/partify>`_ is a web based MPD client focusing on
```

Comment:
For the record, both spellings are technically correct. You normally see single s as it conforms to the general rule (and is also valid American English). But I doubt it matters!